<div class='alert alert-warning'>

SciPy's interactive examples with Jupyterlite are experimental and may not always work as expected. Execution of cells containing imports may result in large downloads (up to 60MB of content for the first import from SciPy). Load times when importing from SciPy may take roughly 10-20 seconds. If you notice any problems, feel free to open an [issue](https://github.com/scipy/scipy/issues/new/choose).

</div>

In the following example, we consider the article "Employee
attitudes and job satisfaction" [3]
which reports the results of a survey from 63 scientists and 117 college
professors. Of the 63 scientists, 31 said they were very satisfied with
their jobs, whereas 74 of the college professors were very satisfied
with their work. Is this significant evidence that college
professors are happier with their work than scientists?
The following table summarizes the data mentioned above
```

college professors   scientists
Very Satisfied   74                     31
Dissatisfied     43                     32

```
When working with statistical hypothesis testing, we usually use a
threshold probability or significance level upon which we decide
to reject the null hypothesis $H_0$. Suppose we choose the common
significance level of 5%.

Our alternative hypothesis is that college professors are truly more
satisfied with their work than scientists. Therefore, we expect
$p_1$ the proportion of very satisfied college professors to be
greater than $p_2$, the proportion of very satisfied scientists.
We thus call `boschloo_exact` with the ``alternative="greater"`` option:


In [ ]:
import scipy.stats as stats
res = stats.boschloo_exact([[74, 31], [43, 32]], alternative="greater")
res.statistic

0.0483

In [ ]:
res.pvalue

0.0355

Under the null hypothesis that scientists are happier in their work than
college professors, the probability of obtaining test
results at least as extreme as the observed data is approximately 3.55%.
Since this p-value is less than our chosen significance level, we have
evidence to reject $H_0$ in favor of the alternative hypothesis.